template/funcs.go

```diff
@@ -472,6 +473,62 @@ func servicesFunc(b *Brain, used, missing *dep.Set) func(...string) ([]*dep.Cata
 	}
 }
 
+// serviceFunc returns or accumulates health service dependencies.
```

Comment:
The comment doesn't match the name of the function.

Reply:
All these methods are so similar that the cut-n-paste urge is high... but that leads to crap like this. :disappointed:

Reply:
Fixed.